Signal/src/ViewControllers/Registration/RegistrationViewController.m

```diff
@@ -566,11 +566,12 @@ - (NSString *_Nullable)debugValueForKey:(NSString *)key
     OWSCAssert(key.length > 0);
 
     NSError *error;
-    NSString *value = [SAMKeychain passwordForService:kKeychainService_LastRegistered account:key error:&error];
-    if (value && !error) {
-        return value;
+    NSString *_Nullable value =
+        [CurrentAppContext().keychainStorage stringForKey:key service:kKeychainService_LastRegistered error:&error];
```

Comment:
That's a good question.  I did this before we settled (if we have) on the new approach of "build all singletons explicitly, in order of dependency, mostly in `AppSetup`, injecting dependencies into initializers, and populate the environments when they're all ready."  

Now that we have that approach, it'd be better to hang the "keychain storage" on an environment.  Let's make that change once the existing work is done in order to avoid rebase hell.